.circleci/config.yml

```diff
@@ -22,7 +22,15 @@ jobs:
       - run: make deps DEPARGS="-v"
       - run: make errcheck
       - run: make check-docs
-      - run: make format
+      - run:
+          name: "Check code formating."
+          command: |
+            make format
+            if ! git diff --exit-code; then
+              echo "Code is not well formatted. Run 'make format' before pushing the code."
+              exit 1
+            fi
+      - run: make proto
```

Comment:
I think we could not run this command, because there is no `protoc` binary in CI container. In my container, I also installed the protocol buffer implementation from https://github.com/google/protobuf and ran `make proto`.

Reply:
true, let's fix this in separate PR